vmm/src/cpu.rs

```diff
@@ -803,9 +814,14 @@ impl CpuManager {
         &self,
         vcpu: Arc<Mutex<Vcpu>>,
         boot_setup: Option<(EntryPoint, &GuestMemoryAtomic<GuestMemoryMmap>)>,
+        #[cfg(feature = "sev_snp")] vmsa_pfn: u64,
```

Comment:
You don't need this, as `vmsa_pfn` can be derived from `boot_setup`.

Reply:
Thanks @likebreath. This is very helpful. Fixed now.